src/test/java/tests/ActionsWithProductsTest.java

```diff
@@ -63,6 +70,8 @@ public void productActionWishListTest() {
         loginPage.inputPasswordField(INPUT_FOR_PASSWORD);
         loginPage.enterButtonClick();
         homePage.navigateToCategoriesFromMainPage();
+        System.out.println(homePage.navigateToLaptopCategory11111());
+        Assert.assertTrue(homePage.navigateToLaptopCategory11111());
```

Comment:
this method returns always true

We cant check that true is true since it's always true...